app/src/main/java/org/simple/clinic/drugs/PrescribedDrug.kt

```diff
@@ -143,5 +143,8 @@ data class PrescribedDrug(
 
     @Query("UPDATE PrescribedDrug SET durationInDays = :durationInDays, updatedAt = :updatedAt WHERE uuid = :id")
     fun updateDrugDuration(id: UUID, durationInDays: Int, updatedAt: Instant)
+
+    @Query("UPDATE PrescribedDrug SET frequency = :drugFrequency, updatedAt = :updatedAt WHERE uuid = :id")
+    fun updateDrugFrequenecy(id: UUID, drugFrequency: MedicineFrequency, updatedAt: Instant)
```

Comment:
Similar to earlier comment about needing to update the sync status of the record.